streams/service/src/main/java/com/hortonworks/streamline/streams/service/SchemaResource.java

```diff
@@ -132,4 +133,61 @@ public Response getKafkaSourceSchema(@PathParam("topicName") String topicName,
         }
     }
 
+    @GET
+    @Path("/{schemaName}/versions")
+    @Timed
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAllSchemaVersions(@PathParam("schemaName") String schemaName,
+                                         @Context SecurityContext securityContext) {
+        try {
+            LOG.info("Get all versions for schema : {}", schemaName);
+            Collection<SchemaVersionInfo> schemaVersionInfos = schemaRegistryClient.getAllVersions(schemaName);
+            Collection<String> schemaVersions = new ArrayList<>();
```

Comment:
Is there any reason for using `Collection<String>` instead of `Collection<Integer>` ?